en/dev_setup/generic_dev_tools.md

```diff
@@ -6,13 +6,24 @@ This section lists useful additional development tools that are not part of the
 
 Download and install the [QGroundControl Daily Build](https://docs.qgroundcontrol.com/en/releases/daily_builds.html).
 
+:::tip
+The *daily build* includes development tools that hidden in release builds. 
+It may also provide access to new PX4 features that are not yet supported in release builds.
+:::
+
 ![QGroundControl](../../assets/toolchain/qgc_goto.jpg)
 
-## Editor / IDE
 
-The development team often use:
+## IDE / Code Editors
+
+The development team recommend [Visual Studio Code (VSCode)](../dev_setup/vscode.md), a popular open source IDE.
```

Comment:
recommend{s}

Typo?

Reply:
Nope, this is correct because the development team are a plural i.e. this is equivalent to:  The manager recommends, the managers recommend.

Reply:
I can certainly learn from you I was assuming it's: One team does whatever.

Reply:
A fair assumption. When I went to school we learned very little grammar, so I basically know what is correct from experience and then have to find a rule that supports my opinion :-)